src/mbgl/map/transform.cpp

```diff
@@ -576,4 +578,14 @@ LatLng Transform::screenCoordinateToLatLng(const ScreenCoordinate& point, LatLng
     return state.screenCoordinateToLatLng(flippedPoint, wrapMode);
 }
 
+double Transform::getMaxPitchForEdgeInsets(const EdgeInsets &insets) const
```

Comment:
How does this method relate to `PITCH_MAX`? Should we incorporate that constant somehow or remove it?

https://github.com/mapbox/mapbox-gl-native/blob/cafecd3f6fedc16e7bf5dff699960978c3b5cb3d/include/mbgl/util/constants.hpp#L37

Reply:
I plan to remove it - which would mean that with no edge inset, getMaxPitchForEdgeInsets would evaluate to 71 degrees, if I remember correctly.

Reply:
> I plan to remove it - which would mean that with no edge inset, getMaxPitchForEdgeInsets would evaluate to 71 degrees, if I remember correctly.

Edit: it is part of work on adaptive tile rendering - the plan is to increase it to e.g. 67.6 deg to prevent pitch limit based on content insets when padding is small